playbooks/vagrant-devstack.yml

```diff
@@ -66,3 +66,4 @@
       when: ENABLE_ELK
     - role: filebeat
       when: ENABLE_ELK
+- include: edx-east/neo4j.yml
```

Comment:
Take a look at what I'm trying over here [1].
I'm wondering if we can simplify this and just add another role to the list, instead of including this entire file. We weren't using nginx (port 80) in testing anyway, right?

[1] https://github.com/stvstnfrd/configuration/pull/19

Reply:
Maybe try it own in the second "no ports" devstack?

Reply:
Do you mean the devstack in the configuration2 directory?